sklearn/tree/_splitter.pyx

```diff
@@ -213,6 +213,14 @@ cdef class Splitter:
 
         return self.criterion.node_impurity()
 
+    cdef int pointer_size(self) nogil:
+        """Get size of a pointer to record for Splitter.
+        
+        Overriding this function allows one to define a 
```

Comment:
truncated sentence.

I suspect you meant "to define a custom tree builder that manage an array of instances of a custom subclass of SplitRecord" or something similar.